create-html-reports.sh

```diff
@@ -154,14 +154,19 @@ if [ "$ALL_REPORTS" != "" ]; then
    " >> $report
     for f in $ALL_REPORTS; do
         report_name=$(basename -s .txt $f)
+        # report_path should be of the form "testing/foo.html"
+        path_to_remove="$RESULTS_DIR/"
```

Comment:
I would have use a more descriptive name like `remove_path_prefix`. At first I though you wanted to delete the directory until I saw down the code that you just wanted to remove a path prefix from a path

Reply:
changed